src/lib/components/wizard.component.spec.ts

```diff
@@ -17,9 +17,11 @@ import { WizardStep } from '../util/wizard-step.interface';
       <aw-wizard-step stepTitle='Steptitle 2'>
         Step 2
       </aw-wizard-step>
-      <aw-wizard-step stepTitle='Steptitle 3' *ngIf="showStep3">
-        Step 3
-      </aw-wizard-step>
+      <div>
+        <aw-wizard-step stepTitle='Steptitle 3' *ngIf="showStep3">
+            Step 3
+        </aw-wizard-step>
+      </div>
```

Comment:
I think it would be create a new dedicated test file instead.
In my opinion this is extended functionality which should not influence the internal functionality tests.
Therefore I think it would be best to create a new test file with two test cases:
- a test case to check whether the component can be created
- a test case to check whether the created wizard has the correct number of steps

Ideally I think the new test setup should consist of multiple `div` containers.

Reply:
To clarify - my fix + test case are addressing an angular breaking change, I don't think it is a "extended functionality". Anyway, I have created the test case as you wish, and i hope you will decide to merge this soon.

Reply:
> To clarify - my fix + test case are addressing an angular breaking change, I don't think it is a "extended functionality".

By extended functionality I mean this is something that the common user of the library does not use. Correct me if I'm wrong but as far as I know the typical user adds his `aw-wizard-step` elements directly to the `aw-wizard` environment.

> Anyway, I have created the test case as you wish, and i hope you will decide to merge this soon.

I'm doing what I can